debezium-schema-generator/src/main/java/io/debezium/schemagenerator/SchemaGenerator.java

```diff
@@ -12,43 +12,68 @@
 import java.util.Optional;
 import java.util.ServiceLoader;
 import java.util.ServiceLoader.Provider;
+import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
-import org.eclipse.microprofile.openapi.models.media.Schema;
-
 import com.google.common.base.Charsets;
 import com.google.common.io.Files;
 
 import io.debezium.metadata.ConnectorMetadata;
 import io.debezium.metadata.ConnectorMetadataProvider;
-import io.debezium.schemagenerator.formats.ApiFormat;
-import io.debezium.schemagenerator.formats.ApiFormatName;
+import io.debezium.schemagenerator.schema.Schema;
+import io.debezium.schemagenerator.schema.SchemaName;
 
 public class SchemaGenerator {
 
+    private static final Logger LOGGER = Logger.getLogger(SchemaGenerator.class.getName());
+
     public static void main(String[] args) {
-        if (args.length != 2) {
-            throw new IllegalArgumentException("Usage: SchemaGenerator <format-name> <output-directory>");
+        if (args.length != 5) {
+            throw new IllegalArgumentException("Usage: SchemaGenerator <format-name> <output-directory> <groupDirectoryPerConnector> <filenamePrefix> <filenameSuffix>");
         }
 
         String formatName = args[0].trim();
         Path outputDirectory = new File(args[1]).toPath();
+        boolean groupDirectoryPerConnector = Boolean.parseBoolean(args[2]);
+        String filenamePrefix = args[3];
+        String filenameSuffix = args[4];
 
-        new SchemaGenerator().run(formatName, outputDirectory);
+        new SchemaGenerator().run(formatName, outputDirectory, groupDirectoryPerConnector, filenamePrefix, filenameSuffix);
     }
 
-    private void run(String formatName, Path outputDirectory) {
+    public void run(String formatName, Path outputDirectory, boolean groupDirectoryPerConnector, String filenamePrefix, String filenameSuffix) {
```

Comment:
Can remain private now?

Reply:
@gunnarmorling Yes, the only case I could imagine if we want to create a Schema Generator CLI or include it in `kcctl` or another Debezium or downstream CLI, then it could make sense to access it as public. wdyt?

Reply:
We can change it to `public` whenever that requirement actually comes up.